nemo_skills/inference/model/base.py

```
@@ -257,53 +261,56 @@ async def generate_async(
+                            )
                     else:
-                        result = self._parse_chat_completion_response(
-                            response, include_response=include_response, **kwargs
-                        )
-                elif endpoint_type == EndpointType.text:
-                    assert isinstance(prompt, str), "Text completion requests must be a string."
-                    request_params = self._build_completion_request_params(prompt=prompt, stream=stream, **kwargs)
-                    response = await litellm.atext_completion(**request_params, **self.litellm_kwargs)
-                    if stream:
-                        result = self._stream_completion_chunks_async(response)
+                        raise TypeError(f"Unsupported completion type: {endpoint_type}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)